src/vcpkg/binarycaching.cpp

```diff
@@ -1449,6 +1449,18 @@ namespace vcpkg
                                                          View<Dependencies::InstallPlanAction> actions)
     {
         auto cache_status = build_cache_status_vector(actions, m_status);
+
+        for (size_t i = 0; i < actions.size(); ++i)
+        {
+            if (cache_status[i] == nullptr)
```

Comment:
Rather than here, I think this should be added to https://github.com/microsoft/vcpkg-tool/pull/331/files#diff-045a3245d6bccf63618bd35376d0a973af7805c6e232bbbf222ca326b60b7000L1422

Ever asking the binary cache about a package without an ABI hash is a bug and should blow up.